ServerCore/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@page
+@model LoginModel
+
+@{
+    ViewData["Title"] = "Log in";
+}
+
+<h2>@ViewData["Title"]</h2>
+<div class="row">
+    <div class="col-md-6 col-md-offset-2">
+        <section>
+            <h4>Use another service to log in.</h4>
+            <hr />
+            @{
+                if ((Model.ExternalLogins?.Count ?? 0) == 0)
+                {
+                    <div>
+                        <p>
+                            Something went wrong! Please contact puzzhunt@microsoft.com to report this error.
```

Comment:
Can you pull the event email for this?

Reply:
I want it to be the general email for now because signing in is independent of events (e.g. imagine people are registering for PD & things are active for PH)